### PR TITLE
kubelet: enable configurable rotation duration and parallel rotate

### PR DESCRIPTION
**Pull request by harshanarayana**
This is a companion PR for the `k/k` item https://github.com/kubernetes/kubernetes/pull/114301

This PR adds a section under the log rotate mechanism to indicate the two new attributes added to the Kubelet's `v1beta1` configuration.

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/logging.md [hunk 1]
```diff
@@ -110,6 +110,13 @@ using the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-co
 These settings let you configure the maximum size for each log file and the maximum number of
 files allowed for each container respectively.
 
+In order to perform an efficient log rotation in clusters where the volume of the logs generated by
+the workload is large, kubelet also provides a mechanism to tune how the logs are rotated in
+terms of how many concurrent log rotations can be performed and the interval at which the logs are
+monitored and rotated as required. These attributes can be configured by setting `containerLogMaxWorkers`
```
Comment (kannon92) [inline content/en/docs/concepts/cluster-administration/logging.md:116]: I'd add that these are kubelet configuration flags.

Look above at rotate and follow the same pattern.

Comment (kannon92) [reply content/en/docs/concepts/cluster-administration/logging.md:116]: > You can configure two kubelet [configuration settings](https://github.com/kubernetes/website/blob/0ca3f289c362ebc1728df80babe4367800a7f8f2/docs/reference/config-api/kubelet-config.v1beta1), containerLogMaxSize and containerLogMaxFiles, using the [kubelet configuration file](https://github.com/kubernetes/website/blob/0ca3f289c362ebc1728df80babe4367800a7f8f2/docs/tasks/administer-cluster/kubelet-config-file). These settings let you configure the maximum size for each log file and the maximum number of files allowed for each container respectively.

Maybe you can follow something similar for these two parameters.

**Comment by reylejano** [comment]
@harshanarayana Is PR https://github.com/kubernetes/kubernetes/pull/114301 , related to the v1.30 release? If so, this PR should target the `dev-1.30` branch

/hold until PR https://github.com/kubernetes/kubernetes/pull/114301 merges and confirmation if this is part of the v1.30 release or not

File: content/en/docs/concepts/cluster-administration/logging.md (hunk 1 above)
Comment (harshanarayana) [reply content/en/docs/concepts/cluster-administration/logging.md:116]: Ack

Comment (harshanarayana) [reply content/en/docs/concepts/cluster-administration/logging.md:116]: @kannon92  should I add a note to indicate this is only added in 1.30 for now ?

**Comment by kannon92** [comment]
/hold cancel
k/k PR merged.

/lgtm 
from technical POV

**Comment by Princesso** [comment]
Hello @kannon92  :wave: please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday March 12th 2024 18:00 PST. Thank you!

**Comment by Princesso** [comment]
Hello @harshanarayana  :wave: please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday March 12th 2024 18:00 PST. Thank you!

**Comment by katcosgrove** [comment]
As the bot has ordained:

/assign @natalisucks

**Pull request merged**